modules/rosa-hcp-sts-creating-a-cluster-cli-no-cni-plugin.adoc

```diff
@@ -69,16 +74,14 @@ The following `State` field changes are listed in the output as the cluster inst
 ====
 If the installation fails or the `State` field does not change to `ready` after more than 10 minutes, check the installation troubleshooting documentation for details. For more information, see _Troubleshooting installations_. For steps to contact Red Hat Support for assistance, see _Getting support for Red Hat OpenShift Service on AWS_.
 ====
-+
-[IMPORTANT]
-====
-When you first log in to the cluster after it reaches `ready` status, the nodes will still be in the `not ready` state until you install your own CNI plugin. After CNI installation, the nodes will change to `ready`. 
-====
 
 . Track the progress of the cluster creation by watching the {product-title} installation program logs. To check the logs, run the following command:
 +
 [source,terminal]
 ----
 $ rosa logs install --cluster=<cluster_name> --watch <1>
 ----
-<1> Optional: To watch for new log messages as the installation progresses, use the `--watch` argument.
\ No newline at end of file
+<1> Optional: To watch for new log messages as the installation progresses, use the `--watch` argument.
+
+== Expected behavior for clusters without a CNI plugin
+Although {hcp-title} cluster installation is complete, the cluster cannot operate without a CNI plugin. For example, the pods do not function, the web console does not operate, and no cluster metrics are available. This is _expected behavior_ until you install a CNI provider.
```

Comment:
I'd say the same thing as follows to align with customer intents.

.... the cluster cannot operate without a CNI plugin. Because the nodes will not be ready, the workloads can not be deployed. For example, the OpenShift cluster console will not be available so to log into the cluster you must use OpenShift CLI (OC); similarly, other OpenShift components such as HA-Proxy based ingress controller, image registry and prometheus-based monitoring stack will not be running.